### ISSUE TITLE
Timesteps of greater than 60 minutes causes error on shortwave dissaggregation

### ISSUE THREAD

**Issue by arbennett** [UW-Hydro/MetSim#12]
This is caused by `tinystep` and `chunk_sum`.

Error message is:

`
````````Traceback (most recent call last):
  File "./ms", line 83, in <module>
    main()
  File "./ms", line 78, in main
    ms.run(setup['method'], setup['forcings'])
  File "/home/bzq/research/MetSim/metsim/metsim.py", line 63, in run
    .to_dataframe(), disagg=True)
  File "/home/bzq/research/MetSim/metsim/methods/mtclim.py", line 37, in run
    forcing = disaggregate(forcing, sg)
  File "/home/bzq/research/MetSim/metsim/disaggregate.py", line 23, in disaggregate
    solar_geom['tiny_rad_fract']))
  File "/home/bzq/research/MetSim/metsim/disaggregate.py", line 106, in shortwave
    chunk_sum(rad[list(tinystep)]) * tmp_rad[day])
ValueError: could not broadcast input array from shape (24) into shape (12)`

**Comment by bartnijssen** [comment]
I think that this works in VIC, since it is routinely run with a 3-hour tilmestep.

**Comment by jhamman** [comment]
Actually, I'm pretty sure this doesn't work in VIC. I think VIC does this step at an hour timestep and then aggregates to the model/snow timestep separately. We probably should support timesteps less than 1 hour but nothing larger for this particular step.

**Comment by bartnijssen** [comment]
OK - then we should reproduce that in MetSim 1.0

**Comment by arbennett** [comment]
Fixed in Fixed in  4b8d37a6
